Description: Fix bold in top manual page
Fix page fault description because storage may not be disk
Author: Craig Small <csmall@debian.org>
Index: b/top.1
===================================================================
--- a/top.1	2009-11-24 20:53:04.000000000 +1100
+++ b/top.1	2009-11-24 21:00:38.000000000 +1100
@@ -42,7 +42,7 @@
 .ds EM \ \fB\-\-\ \fR
 \#                          - these two are for chuckles, makes great grammar
 .ds Me top
-.ds ME \fBtop\fR
+.ds NE \fBtop\fR
 \#                          - other misc strings for consistent usage/emphasis
 .ds F \fIOff\fR
 .ds O \fIOn\fR
@@ -85,7 +85,7 @@
 .\" ----------------------------------------------------------------------
 .SH SYNOPSIS
 .\" ----------------------------------------------------------------------
-\*(ME \-\fBhv\fR | \-\fBbcHisS\fR \-\fBd\fI delay\fR \-\fBn\fI
+\*(NE \-\fBhv\fR | \-\fBbcHisS\fR \-\fBd\fI delay\fR \-\fBn\fI
 iterations\fR \-\fBp\fI pid\fR [,\fI pid\fR ...]
 
 The traditional switches '-' and whitespace are optional.
@@ -94,7 +94,7 @@
 .\" ----------------------------------------------------------------------
 .SH DESCRIPTION
 .\" ----------------------------------------------------------------------
-The \*(ME program provides a dynamic real-time view of a running system.
+The \*(NE program provides a dynamic real-time view of a running system.
 It can display\fB system\fR summary information as well as a list of\fB
 tasks\fR currently being managed by the Linux kernel.
 The types of system summary information shown and the types, order and
@@ -104,7 +104,7 @@
 The program provides a limited interactive interface for process
 manipulation as well as a much more extensive interface for personal
 configuration \*(EM encompassing every aspect of its operation.
-And while \*(ME is referred to throughout this document, you are free
+And while \*(NE is referred to throughout this document, you are free
 to name the program anything you wish.
 That new name, possibly an alias, will then be reflected on \*(Me's display
 and used when reading and writing a \*(CF.
@@ -190,7 +190,7 @@
        'I' - Irix mode        On\ \ (no, 'solaris' smp)
      * 'p' - PID monitoring   Off
      * 's' - Secure mode      Off (unsecured)
-       'B' - Bold disable     Off
+       'B' - Bold enable      Off
     \fISummary_Area_defaults\fR
        'l' - Load Avg/Uptime  On\ \ (thus program name)
        't' - Task/Cpu states  On\ \ (1+1 lines, see '1')
@@ -413,16 +413,14 @@
 q:\fB RES\fR \*(EM Resident size (kb)
 The non-swapped \*(MP a task has used.
 
-RES = CODE + DATA.
-
 .TP 3
 r:\fB CODE\fR \*(EM Code size (kb)
-The amount of \*(MP devoted to executable code, also known as
+The amount of \*(MV devoted to executable code, also known as
 the 'text resident set' size or TRS.
 
 .TP 3
 s:\fB DATA\fR \*(EM Data+Stack size (kb)
-The amount of \*(MP devoted to other than executable code, also known as
+The amount of \*(MV devoted to other than executable code, also known as
 the 'data resident set' size or DRS.
 
 .TP 3
@@ -436,8 +434,8 @@
 The number of\fB major\fR page faults that have occurred for a task.
 A page fault occurs when a process attempts to read from or write to a virtual
 page that is not currently present in its address space.
-A major page fault is when disk access is involved in making that
-page available.
+A major page fault is when backing storage access (such as a disk) is involved
+in making that page available.
 
 .TP 3
 v:\fB nDRT\fR \*(EM Dirty Pages count
@@ -504,7 +502,7 @@
 .\" ......................................................................
 .SS 2b. SELECTING and ORDERING Columns
 .\" ----------------------------------------------------------------------
-After pressing the \*(CIs 'f' (Fields select) or \'o' (Order fields) you will
+After pressing the \*(CIs 'f' (Fields select) or 'o' (Order fields) you will
 be shown a screen containing the current \fBfields string\fR followed by names
 and descriptions for all fields.
 
@@ -580,7 +578,7 @@
 see current status,
 
 .TP 7
-\ \ \'\fB?\fR\' or \'\fBh\fR\' :\fIHelp\fR
+\ \ \<\fB?\fR\> or \<\fBh\fR\> :\fIHelp\fR
 There are two help levels available.
 The first will provide a reminder of all the basic \*(CIs.
 If \*(Me is\fI secured\fR, that screen will be abbreviated.
@@ -589,7 +587,7 @@
 applicable to \*(AM.
 
 .TP 7
-\ \ \'\fB=\fR\' :\fIExit_Task_Limits\fR
+\ \ \<\fB=\fR\> :\fIExit_Task_Limits\fR
 Removes restrictions on which tasks are shown.
 This command will reverse any 'i' (idle tasks) and 'n' (max tasks) commands
 that might be active.
@@ -599,13 +597,13 @@
 When operating in \*(AM this command has a slightly broader meaning.
 
 .TP 7
-\ \ \'\fBA\fR\' :\fIAlternate_Display_Mode_toggle\fR
+\ \ \<\fBA\fR\> :\fIAlternate_Display_Mode_toggle\fR
 This command will switch between \*(FM and \*(AM.
-\*(XT 4. ALTERNATE\-DISPLAY Mode and the 'G' \*(CI for insight into
-\*(CWs and field groups.
+\*(XT 4. ALTERNATE\-DISPLAY Mode and the 'G' \*(CI for insight
+into \*(CWs and field groups.
 
 .TP 7
-\ \ \'\fBB\fR\' :\fIBold_Disable/Enable_toggle\fR
+\ \ \<\fBB\fR\> :\fIBold_Disable/Enable_toggle\fR
 This command will influence use of the 'bold' terminfo capability and
 alters\fB both\fR the \*(SA and \*(TA for the \*(CW.
 While it is intended primarily for use with dumb terminals, it can be
@@ -617,7 +615,7 @@
 there will be no visual confirmation that they are even on.
 
 .TP 7
-*\ \'\fBd\fR\' or \'\fBs\fR\' :\fIChange_Delay_Time_interval\fR
+*\ \<\fBd\fR\> or \<\fBs\fR\> :\fIChange_Delay_Time_interval\fR
 You will be prompted to enter the delay time, in seconds, between
 display updates.
 
@@ -631,32 +629,32 @@
 and view the system summary on the second line.
 
 .TP 7
-\ \ \'\fBG\fR\' :\fIChoose_Another_Window/Field_Group\fR
+\ \ \<\fBG\fR\> :\fIChoose_Another_Window/Field_Group\fR
 You will be prompted to enter a number between 1 and 4 designating the
 window/field group which should be made the \*(CW.
 You will soon grow comfortable with these 4 windows, especially after
 experimenting with \*(AM.
 
 .TP 7
-\ \ \'\fBI\fR\' :\fIIrix/Solaris_Mode_toggle\fR
+\ \ \<\fBI\fR\> :\fIIrix/Solaris_Mode_toggle\fR
 When operating in 'Solaris mode' ('I' toggled \*F), a task's \*(Pu usage
 will be divided by the total number of \*(PUs.
 After issuing this command, you'll be informed of the new state of this toggle.
 
 .TP 7
-\ \ \'\fBu\fR\' :\fIselect a user\fR
+\ \ \<\fBu\fR\> :\fIselect a user\fR
 You will be prompted for a UID or username. Only processes
 belonging to the selected user will be displayed. This option
 matches on the effective UID.
 
 .TP 7
-\ \ \'\fBU\fR\' :\fIselect a user\fR
+\ \ \<\fBU\fR\> :\fIselect a user\fR
 You will be prompted for a UID or username. Only processes
 belonging to the selected user will be displayed. This option
 matches on the real, effective, saved, and filesystem UID.
 
 .TP 7
-*\ \'\fBk\fR\' :\fIKill_a_task\fR
+*\ \<\fBk\fR\> :\fIKill_a_task\fR
 You will be prompted for a PID and then the signal to send.
 The default signal, as reflected in the prompt, is SIGTERM.
 However, you can send any signal, via number or name.
@@ -667,24 +665,24 @@
    2) at the signal prompt, type 0
 
 .TP 7
-\ \ \'\fBq\fR\' :\fIQuit\fR
+\ \ \<\fBq\fR\> :\fIQuit\fR
 
 .TP 7
-*\ \'\fBr\fR\' :\fIRenice_a_Task\fR
+*\ \<\fBr\fR\> :\fIRenice_a_Task\fR
 You will be prompted for a PID and then the value to nice it to.
 Entering a positive value will cause a process to lose priority.
 Conversely, a negative value will cause a process to be viewed more
 favorably by the kernel.
 
 .TP 7
-\ \ \'\fBW\fR\' :\fIWrite_the_Configuration_File\fR
+\ \ \<\fBW\fR\> :\fIWrite_the_Configuration_File\fR
 This will save all of your options and toggles plus the current
 display mode and delay time.
 By issuing this command just before quitting \*(Me, you will be able restart
 later in exactly that same state.
 
 .TP 7
-\ \ \'\fBZ\fR\' :\fIChange_Color_Mapping
+\ \ \<\fBZ\fR\> :\fIChange_Color_Mapping
 This key will take you to a separate screen where you can change the
 colors for the \*(CW, or for all windows.
 For details regarding this \*(CI \*(Xt 3d. COLOR Mapping.
@@ -700,25 +698,25 @@
 of messages and prompts.
 
 These commands always impact just the \*(CW/field group.
-\*(XT 4. ALTERNATE\-DISPLAY Mode and the 'G' \*(CI for insight into
-\*(CWs and field groups.
+\*(XT 4. ALTERNATE\-DISPLAY Mode and the 'G' \*(CI for insight
+into \*(CWs and field groups.
 
 .TP 7
-\ \ \'\fBl\fR\' :\fIToggle_Load_Average/Uptime\fR \*(EM On/Off
+\ \ \<\fBl\fR\> :\fIToggle_Load_Average/Uptime\fR \*(EM On/Off
 This is also the line containing the program name (possibly an alias) when
 operating in \*(FM or the \*(CW name when operating in \*(AM.
 
 .TP 7
-\ \ \'\fBm\fR\' :\fIToggle_Memory/Swap_Usage\fR \*(EM On/Off
+\ \ \<\fBm\fR\> :\fIToggle_Memory/Swap_Usage\fR \*(EM On/Off
 This command affects two \*(SA lines.
 
 .TP 7
-\ \ \'\fBt\fR\' :\fIToggle_Task/Cpu_States\fR \*(EM On/Off
+\ \ \<\fBt\fR\> :\fIToggle_Task/Cpu_States\fR \*(EM On/Off
 This command affects from 2 to many \*(SA lines, depending on the state
 of the '1' toggle and whether or not \*(Me is running under true SMP.
 
 .TP 7
-\ \ \'\fB1\fR\' :\fIToggle_Single/Separate_Cpu_States\fR \*(EM On/Off
+\ \ \<\fB1\fR\> :\fIToggle_Single/Separate_Cpu_States\fR \*(EM On/Off
 This command affects how the 't' command's Cpu States portion is shown.
 Although this toggle exists primarily to serve massively-parallel SMP machines,
 it is not restricted to solely SMP environments.
@@ -750,12 +748,12 @@
 .in
 
 .TP 7
-\ \ \'\fBb\fR\' :\fIBold/Reverse_toggle\fR
+\ \ \<\fBb\fR\> :\fIBold/Reverse_toggle\fR
 This command will impact how the 'x' and 'y' toggles are displayed.
 Further, it will only be available when at least one of those toggles is \*O.
 
 .TP 7
-\ \ \'\fBx\fR\' :\fIColumn_Highlight_toggle\fR
+\ \ \<\fBx\fR\> :\fIColumn_Highlight_toggle\fR
 Changes highlighting for the current sort field.
 You probably don't need a constant visual reminder of the sort field and
 \*(Me hopes that you always run with 'column highlight' \*F, due to the cost
@@ -765,7 +763,7 @@
 visual reminder.
 
 .TP 7
-\ \ \'\fBy\fR\' :\fIRow_Highlight_toggle\fR
+\ \ \<\fBy\fR\> :\fIRow_Highlight_toggle\fR
 Changes highlighting for "running" tasks.
 For additional insight into this task state, \*(Xt 2a. DESCRIPTIONS of Fields,
 Process Status.
@@ -774,7 +772,7 @@
 The only costs will be a few additional tty escape sequences.
 
 .TP 7
-\ \ \'\fBz\fR\' :\fIColor/Monochrome_toggle\fR
+\ \ \<\fBz\fR\> :\fIColor/Monochrome_toggle\fR
 Switches the \*(CW between your last used color scheme and the older form
 of black-on-white or white-on-black.
 This command will alter\fB both\fR the \*(SA and \*(TA but does not affect the
@@ -785,20 +783,20 @@
 .B CONTENT\fR of \*(TW
 .PD 0
 .TP 7
-\ \ \'\fBc\fR\' :\fICommand_Line/Program_Name_toggle\fR
+\ \ \<\fBc\fR\> :\fICommand_Line/Program_Name_toggle\fR
 This command will be honored whether or not the 'Command' column
 is currently visible.
 Later, should that field come into view, the change you applied will be seen.
 
 .TP 7
-\ \ \'\fBf\fR\' and \'\fBo\fR\' :\fIFields_select\fR or \fIOrder_fields\fR
+\ \ \<\fBf\fR\> and \<\fBo\fR\> :\fIFields_select\fR or \fIOrder_fields\fR
 These keys display separate screens where you can change which
 fields are displayed and their order.
 For additional information on these \*(CIs
 \*(Xt 2b. SELECTING and ORDERING Columns.
 
 .TP 7
-\ \ \'\fBH\fR\' :\fIThreads_toggle\fR
+\ \ \<\fBS\fR\> :\fICumulative_Time_Mode_toggle\fR
 When this toggle is \*O, all individual threads will be displayed.  Otherwise, \*(Me displays a summation of all threads in a process.
 
 .TP 7
@@ -818,7 +816,7 @@
 effect, simply ask for help and view the window summary on the second line.
 
 .TP 7
-\ \ \'\fBu\fR\' :\fIShow_Specific_User_Only\fR
+\ \ \<\fBu\fR\> :\fIShow_Specific_User_Only\fR
 You will be prompted to enter the name of the user to display.
 Thereafter, in that \*(TW only matching User ID's will be shown, or possibly
 no tasks will be shown.
@@ -831,7 +829,7 @@
 .B SIZE\fR of \*(TW
 .PD 0
 .TP 7
-\ \ \'\fBi\fR\' :\fIIdle_Processes_toggle\fR
+\ \ \<\fBi\fR\> :\fIIdle_Processes_toggle\fR
 Displays all tasks or just active tasks.
 When this toggle is \*F, idled or zombied processes will not be displayed.
 
@@ -839,7 +837,7 @@
 affect the window's size, as all prior \*(TDs will have already been painted.
 
 .TP 7
-\ \ \'\fBn\fR\' or \'#\' :\fISet_Maximum_Tasks\fR
+\ \ \<\fBn\fR\> or \<\fB#\fR\> :\fISet_Maximum_Tasks\fR
 You will be prompted to enter the number of tasks to display.
 The lessor of your number and available screen rows will be used.
 
@@ -878,12 +876,12 @@
 .in
 
 .TP 7
-\ \ \'\fB<\fR\' :\fIMove_Sort_Field_Left\fR
+\ \ \<\fB<\fR\> :\fIMove_Sort_Field_Left\fR
 Moves the sort column to the left unless the current sort field is
 the first field being displayed.
 
 .TP 7
-\ \ \'\fB>\fR\' :\fIMove_Sort_Field_Right\fR
+\ \ \<\fB>\fR\> :\fIMove_Sort_Field_Right\fR
 Moves the sort column to the right unless the current sort field is
 the last field being displayed.
 
@@ -894,7 +892,7 @@
 .in
 
 .TP 7
-\ \ \'\fBF\fR\' or \'\fBO\fR\' :\fISelect_Sort_Field\fR
+\ \ \<\fBF\fR\> or \<\fBO\fR\> :\fISelect_Sort_Field\fR
 These keys display a separate screen where you can change which field
 is used as the sort column.
 
@@ -907,7 +905,7 @@
 when running \*(Me with column highlighting turned \*F.
 
 .TP 7
-\ \ \'\fBR\fR\' :\fIReverse/Normal_Sort_Field_toggle\fR
+\ \ \<\fBR\fR\> :\fIReverse/Normal_Sort_Field_toggle\fR
 Using this \*(CI you can alternate between high-to-low and low-to-high sorts.
 
 .PP
@@ -984,7 +982,7 @@
 .\" ......................................................................
 .SS 4b. COMMANDS for Windows
 .TP 7
-\ \ \'\fB-\fR\' and \'\fB_\fR\' :\fIShow/Hide_Window(s)_toggles\fR
+\ \ \<\fB-\fR\> and \<\fB_\fR\> :\fIShow/Hide_Window(s)_toggles\fR
 The '-' key turns the \*(CW's \*(TD \*O and \*F.
 When \*O, that \*(TA will show a minimum of the columns header you've
 established with the 'f' and 'o' commands.
@@ -998,7 +996,7 @@
 as the only display element.
 
 .TP 7
-*\ \'\fB=\fR\' and \'\fB+\fR\' :\fIEqualize_(re-balance)_Window(s)\fR
+*\ \<\fB=\fR\> and \<\fB+\fR\> :\fIEqualize_(re-balance)_Window(s)\fR
 The '=' key forces the \*(CW's \*(TD to be visible.
 It also reverses any 'i' (idle tasks) and 'n' (max tasks) commands that might
 be active.
@@ -1009,7 +1007,7 @@
 except for the 'i' (idle tasks) and 'n' (max tasks) commands.
 
 .TP 7
-*\ \'\fBA\fR\' :\fIAlternate_Display_Mode_toggle\fR
+*\ \<\fBA\fR\> :\fIAlternate_Display_Mode_toggle\fR
 This command will switch between \*(FM and \*(AM.
 
 The first time you issue this command, all four \*(TDs will be shown.
@@ -1017,7 +1015,7 @@
 chosen to make visible.
 
 .TP 7
-*\ \'\fBa\fR\' and \'\fBw\fR\' :\fINext_Window_Forward/Backward\fR
+*\ \<\fBa\fR\> and \<\fBw\fR\> :\fINext_Window_Forward/Backward\fR
 This will change the \*(CW, which in turn changes the window to which
 commands are directed.
 These keys act in a circular fashion so you can reach any desired \*(CW
@@ -1028,7 +1026,7 @@
 the \*(TD is \*F and many commands will be restricted.
 
 .TP 7
-*\ \'\fBG\fR\' :\fIChoose_Another_Window/Field_Group\fR
+*\ \<\fBG\fR\> :\fIChoose_Another_Window/Field_Group\fR
 You will be prompted to enter a number between 1 and 4 designating the
 window/field group which should be made the \*(CW.
 
@@ -1037,15 +1035,15 @@
 commands.
 
 .TP 7
-\ \ \'\fBg\fR\' :\fIChange_Window/Field_Group_Name\fR
+\ \ \<\fBg\fR\> :\fIChange_Window/Field_Group_Name\fR
 You will be prompted for a new name to be applied to the \*(CW.
 It does not require that the window name be visible
 (the 'l' toggle to be \*O).
 
 .IP "*" 3
 The \*(CIs shown with an \*(AS have use beyond \*(AM.
-    \'=', 'A', 'G'  are always available
-    \'a', 'w'       act the same when color mapping
+    '=', 'A', 'G'  are always available
+    'a', 'w'       act the same when color mapping
 
 
 .\" ----------------------------------------------------------------------
